src/input_graph.rs

```diff
@@ -206,6 +202,17 @@ impl InputGraph {
         g.freeze();
         g
     }
+
+    /// Writes the input graph to a text file, using the following format:
+    /// a <from> <to> <weight>
+    /// Mostly used for performance testing.
+    pub fn write_to_file(&self, filename: &str) -> Result<(), std::io::Error> {
```

Comment:
I would go for either `write_to_file/read_from_file` or `from_file/to_file`

Reply:
Done, `from_file/to_file`